cmd/uplink/cmd/cp.go

```diff
@@ -34,6 +36,7 @@ func init() {
 		RunE:  copyMain,
 	}, RootCmd)
 	progress = cpCmd.Flags().Bool("progress", true, "if true, show progress")
+	ttl = cpCmd.Flags().Duration("ttl", 0, "number of days before file deletion. Default (0) is indefinite")
```

Comment:
Maybe we can name it `expiration` or something like that?

Reply:
or `days-to-live`, or `expires-in`.